### PR TITLE
Selfie: Filter out unsupported video qualities

### PR DESCRIPTION
**Pull request by luk1337**
Change-Id: I6034726ffcba55fc9b07208cb4157f59539432c2

### CODE REVIEW

File: app/src/main/java/org/lineageos/selfie/SettingsActivity.kt [hunk 1]
```diff
@@ -75,6 +80,29 @@ class SettingsActivity : AppCompatActivity() {
...
+                }
+                it.entries = map.keys.toTypedArray()
+                it.entryValues = map.values.toTypedArray()
+            }
+        }
+
+        @androidx.camera.camera2.interop.ExperimentalCamera2Interop
+        private fun supportedVideoQualities(): List<Quality> {
+            val cameraProvider = ProcessCameraProvider.getInstance(requireContext()).get()
+            val cameraInfo = cameraProvider.availableCameraInfos[0]
```
Comment (SebaUbuntu) [inline app/src/main/java/org/lineageos/selfie/SettingsActivity.kt]: Every camera has different video qualities

File: app/src/main/java/org/lineageos/selfie/SettingsActivity.kt [hunk 2]
```diff
@@ -75,6 +80,29 @@ class SettingsActivity : AppCompatActivity() {
...
+                        }
+                    )
+                }
+                it.entries = map.keys.toTypedArray()
+                it.entryValues = map.values.toTypedArray()
+            }
+        }
+
+        @androidx.camera.camera2.interop.ExperimentalCamera2Interop
+        private fun supportedVideoQualities(): List<Quality> {
```
Comment (SebaUbuntu) [inline app/src/main/java/org/lineageos/selfie/SettingsActivity.kt]: Can be a PhysicalCamera method

File: app/src/main/java/org/lineageos/selfie/SettingsActivity.kt (hunk 1 above)
Comment (luk1337) [reply app/src/main/java/org/lineageos/selfie/SettingsActivity.kt]: Yes, aware of that. I suppose we can reuse this later to add separate back facing camera / front facing camera video quality pref.

**Comment by luca020400** [comment]
Can't we move the setting to the "top bar"?

File: app/src/main/java/org/lineageos/selfie/SettingsActivity.kt (hunk 2 above)
Comment (luk1337) [reply app/src/main/java/org/lineageos/selfie/SettingsActivity.kt]: Done.

**Comment by SebaUbuntu** [comment]
Alternative: https://github.com/SebaUbuntu/android_packages_apps_Selfie/pull/78

**Pull request closed without merging**